### PR TITLE
Creating a Report for the conversion  

### PR DESCRIPTION
**Pull request by Peyman-N**

### CODE REVIEW

File: bids2openminds/report.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import os
+
+
+def create_report(dataset, dataset_version, collection, dataset_description, input_path):
+    print(f"Dataset title : {dataset.full_name}")
```
Comment (lzehl) [inline bids2openminds/report.py:5]: you cannot just print the individual elements, but have to write/collect them first into an object that is then printed by executing the function (with option to set to quite) and presented as an output of that function in addition to the collection.

Comment (lzehl) [reply bids2openminds/report.py:5]:
```
Conversion Report
=================

The following elements were converted:
--------------------------------------
+ number of converted subjects: XX
+ number of state for each subject: XX-YY
+ number of files: XX

The following elements were not converted:
------------------------------------------
+ derivative data are ignored for now
+ the following suffixes were not translated to techniques yet: [x,y,z]
```

Comment (lzehl) [reply bids2openminds/report.py:5]: (this is an example if you decide for a markdown text file, keep the first version simple we can always add more information later on)

**Review by lzehl — CHANGES_REQUESTED** [review]

**Comment by Peyman-N** [comment]
@lzehl I modified it as per demand. 
The number of subject states for each subject is problematic, as it can vary from subject to subject. What are your thoughts on this: should we present the average number, the total number of subjects, or perhaps a list of subjects with their respective number of states?
Here is a example of report file.
Screenshot from 2024-06-05 14-47-45

**Comment by Peyman-N** [comment]
It should have a sentence specifying the authors should double checked, and list all the authors as they have been extracted.

**Pull request merged**